Can't println char array

**V version:1.26** 
**OS:macos 10.15.4** 

**What did you do?**
```
fn main() {
	chars := [`a`, `b`] 
	println(chars)
}
```

**What did you expect to see?**
 [`a`, `b`] 
 
**What did you see instead?**
```cannot generate str() for array_byte
==================
^
/var/folders/pg/rhbyz_t94c5fqcblmkxp6cyh0000gn/T/v/test.tmp.c:4629:10: error: implicit declaration of function 'array_byte_str' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
        println(array_byte_str(chars));
                ^
/var/folders/pg/rhbyz_t94c5fqcblmkxp6cyh0000gn/T/v/test.tmp.c:4629:10: error: passing 'int' to parameter of incompatible type 'string' (aka 'struct string')
        println(array_byte_str(chars));
                ^~~~~~~~~~~~~~~~~~~~~
/var/folders/pg/rhbyz_t94c5fqcblmkxp6cyh0000gn/T/v/test.tmp.c:1742:21: note: passing argument to parameter 's' here
void println(string s) {
                    ^
/var/folders/pg/rhbyz_t94c5fqcblmkxp6cyh0000gn/T/v/test.tmp.c:4995:4: warning: initializer overrides prior initialization of this subobject [-Winitializer-overrides]
                .u = 0,
...
==================
(Use `v -cg` to print the entire error message)

builder error: C error.```

--- a/vlib/builtin/array.v
+++ b/vlib/builtin/array.v
@@ -373,7 +373,7 @@ pub fn (a []string) str() string {
 }
 
 // []byte.str returns a string representation of the array of bytes
-// => '[a, b, c]'
+// => '[`a`, `b`, `c`]'
 pub fn (a []byte) str() string {
 	mut sb := strings.new_builder(a.len * 3)
 	sb.write('[')

--- a/vlib/builtin/array.v
+++ b/vlib/builtin/array.v
@@ -372,6 +372,27 @@ pub fn (a []string) str() string {
 	return sb.str()
 }
 
+// []byte.str returns a string representation of the array of bytes
+// => '[a, b, c]'
+pub fn (a []byte) str() string {
+	mut sb := strings.new_builder(a.len * 3)
+	sb.write('[')
+	for i in 0..a.len {
+		val := a[i].str()
+		sb.write('`')
+		sb.write(val)
+		sb.write('`')
+		if a[i] != 0 {
+			val.free()
+		}
+		if i < a.len - 1 {
+			sb.write(', ')
+		}
+	}
+	sb.write(']')
+	return sb.str()
+}
+
 // []int.str returns a string representation of the array of ints
 // => '[1, 2, 3]'
 pub fn (a []int) str() string {

